ci/benchmarks/partial-conv/evo2_pretrain.yaml

```diff
@@ -16,7 +16,7 @@ script_args:
   workspace: /workspace/bionemo2
   data_path: /data/evo2
   artefacts_url: https://__token__:${JET_GITLAB_TOKEN}@gitlab-master.nvidia.com/api/v4/projects/180496/packages/pypi/simple
-  file_name_wheel: cuhyena==v0.2.3+cuda12.9
+  file_name_wheel: subquadratic-ops
```

Comment:
why no versions anymore? Just double checking this is intentional.

Reply:
There is a version `0.0.1+cuda12.9` Let's add it here. we can track version impacts.